woocommerce/woocommerce-ios
# Reapply updates from "item price" PR (#577)

This PR is a dupe of #577 — it re-adds changes from that.

## Testing

- [ ] Build and run all the unit tests — verify everything is ✅ 

Here are some suggested testing steps, but feel free to test this however you want.

1. Build and run the app
2. Open the orders tab
3. Find an order with **`Processing` status** where the line items have a **quantity of 1**
4. Open the fullfillment screen
    - [ ] Verify the item price and quantity details looks good
5. Find an order with **`Completed` status** where the line items have a **quantity of 1**
6. Open the item details screen
    - [ ] Verify the item price and quantity details looks good

7. Find an order with **`Processing` status** where the line items have a **quantity of 2 or more**
8. Open the fullfillment screen
    - [ ] Verify the item price and quantity details looks good
9. Find an order with **`Completed` status** where the line items have a **quantity of 2 or more**
10. Open the item details screen
    - [ ] Verify the item price and quantity details looks good
